android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java

```diff
@@ -39,17 +42,24 @@
 public class JitsiMeetOngoingConferenceService extends Service
     implements OngoingConferenceTracker.OngoingConferenceListener {
     private static final String TAG = JitsiMeetOngoingConferenceService.class.getSimpleName();
+    private static final String EXTRA_DATA_KEY = "extraDataKey";
+    private static final String EXTRA_DATA_BUNDLE_KEY = "extraDataBundleKey";
+    private static final String IS_AUDIO_MUTED_KEY = "isAudioMuted";
 
     private final BroadcastReceiver broadcastReceiver = new BroadcastReceiver();
 
     private boolean isAudioMuted;
 
-    static void launch(Context context) {
+    static void launch(Context context, HashMap<String, Object> extraData) {
```

Comment:
Shouldn't we keep the previous one too, for backwards compat?

Reply:
Not sure whether there could be a situation where this requires backwards compat. Do you think there might be users that use this service on other ways that we use it?

Reply:
Maybe if they don't use our Activity? This is part of the public API, isn't it?